### DISCUSSION TITLE
NetTopologySuite.IO.GeoTools compatibility with NetTopologySuite new versions

### DISCUSSION THREAD

**Discussion by pratibha-gaglani** [NetTopologySuite/NetTopologySuite#593] [Q&A]
NetTopologySuite.IO.GeoTools has NOT seen an upgrade since 1.14 V, and the libraries are unfortunately not compatible with the new NetTopologySuite 2x versions.

I am particularly interested in the ShapeFileDataReader class and after upgrading the rest of NetTopologySuite, I am unable to use the ShapeFileDataReader from NetTopologySuite.IO.GeoTools, because of GeoAPI references. 

Any thoughts or inputs on the topic will be appreciated. 

Thanks

**Comment by DGuidi** [comment]
duplicated, see #592
